rmgpy/reaction.py

```diff
@@ -677,7 +679,7 @@ def get_rate_coefficient(self, T, P=0):
         else:
             return self.kinetics.get_rate_coefficient(T, P)
 
-    def get_surface_rate_coefficient(self, T, surface_site_density):
+    def get_surface_rate_coefficient(self, T, surface_site_density=2.5e-05):
```

Comment:
I don't think this one is required, because it's set by the calling functions?  I'm not very happy with having a default, in general, as it increases the chance of someone forgetting to pass it when they should.  (There's no current bug, I'm just anticipating future bugs..)